contrib/opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -1125,6 +1130,63 @@ def test_aaf_marker_over_transition(self):
 
         self.assertIsNotNone(timeline)
 
+    def test_aaf_marker_over_audio_file(self):
+        """
+        Make sure we can transcibe markers over an audio AAF file.
+        """
+
+        timeline = None
+
+        try:
+            timeline = otio.adapters.read_from_file(
+                MARKER_OVER_AUDIO_PATH
+            )
+
+        except Exception as e:
+            print('[ERROR] Transcribing test sample data `{}` caused an exception: {}'.format(  # noqa
+                os.path.basename(MARKER_OVER_AUDIO_PATH),
+                e)
+            )
+
+        self.assertIsNotNone(timeline)
+
+        # Verify markers
+        # We expect 1 track with 3 markers on it from the test data.
+        self.assertTrue(1 == len(timeline.tracks))
```

Comment:
`assertEqual` is a better choice, since any failure will be reported more clearly.

Reply:
Switched to assertEqual.